### PR TITLE
Add model's classes

### PR DESCRIPTION
**Pull request by toxa055**

### CODE REVIEW

File: src/main/java/com/rehab/model/Cure.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.rehab.model;
+
+public class Cure extends AbstractNamedEntity {
+    private CureType cureType;
+
+    public Cure() {
```
Comment (shapuran) [inline src/main/java/com/rehab/model/Cure.java:6]: When you define default constructor with empty body then it is possible that getCureType returns NULL. You should either get rid of this constructor or initialize cureType with some default value.

File: src/main/java/com/rehab/model/Patient.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.rehab.model;
+
+public class Patient extends AbstractNamedEntity {
+    private int insuranceNumber;
+    private Doctor doctor;
+    private PatientState patientState;
+    private String diagnosis;
+
+    public Patient() {
```
Comment (shapuran) [inline src/main/java/com/rehab/model/Patient.java:9]: The same as with Cure class.

**Review by shapuran — CHANGES_REQUESTED** [review]

File: src/main/java/com/rehab/model/Cure.java (hunk 1 above)
Comment (toxa055) [reply src/main/java/com/rehab/model/Cure.java:6]: Initialized cureType with MEDICINE in commit c04ab95

File: src/main/java/com/rehab/model/Patient.java (hunk 2 above)
Comment (toxa055) [reply src/main/java/com/rehab/model/Patient.java:9]: Initialized patientState with TREATING in c04ab95

File: src/main/java/com/rehab/model/Admin.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.rehab.model;
+
+public class Admin extends AbstractEmployee {
+
+    public Admin() {
```
Comment (shapuran) [inline src/main/java/com/rehab/model/Admin.java:5]: For example when I initialize:
    Admin admin = new Admin();
What will be when:
   admin.getEmail()
   admin.getPassword()
   admin.getRole()
?

**Review by shapuran — CHANGES_REQUESTED** [review]

File: src/main/java/com/rehab/model/Admin.java (hunk 3 above)
Comment (toxa055) [reply src/main/java/com/rehab/model/Admin.java:5]: null will be returned. I created constructor with no args, because it is supposed to be for JPA.

**Review by shapuran — APPROVED** [review]

**Pull request merged**